src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+
+    const changeToLowerCase = (word: string) => {
```

Comment:
title, description, and query are strings, so you can use .toLowerCase(), and not to make a special function

Reply:
good job!